mne/channels/tests/test_montage.py

```diff
@@ -836,26 +889,20 @@ def test_fif_dig_montage():
 @testing.requires_testing_data
 def test_egi_dig_montage():
     """Test EGI MFF XML dig montage support."""
-    with pytest.deprecated_call():
-        dig_montage = read_dig_montage(egi=egi_dig_montage_fname, unit='m')
-
-    # # test round-trip IO
-    temp_dir = _TempDir()
-    fname_temp = op.join(temp_dir, 'egi_test.fif')
-    _check_roundtrip(dig_montage, fname_temp)
-
-    with pytest.deprecated_call():
-        # Test coordinate transform
-        # dig_montage.transform_to_head()  # XXX: this call had no effect!!
-        # nasion
-        assert_almost_equal(dig_montage.nasion[0], 0)
-        assert_almost_equal(dig_montage.nasion[2], 0)
-        # lpa and rpa
-        assert_allclose(dig_montage.lpa[1:], 0, atol=1e-16)
-        assert_allclose(dig_montage.rpa[1:], 0, atol=1e-16)
+    dig_montage = read_dig_egi(egi_dig_montage_fname)
+    fid, coord = _get_fid_coords(dig_montage.dig)
+
+    assert coord == FIFF.FIFFV_COORD_UNKNOWN
+    assert_allclose(
+        actual=np.array([vv for vv in fid.values()]),
```

Comment:
dict iteration order not guaranteed, iterate over `['lpa', 'nasion', 'rpa']` instead

Reply:
yep. This one again.